sdk/identity/azure-identity/tests/test_vscode_credential.py

```diff
@@ -67,14 +67,11 @@ def test_get_token():
         assert mock_client.obtain_token_by_refresh_token.call_count == 1
 
 
-if sys.platform.startswith('win'):
-    from azure.identity._credentials.win_vscode_adapter import _read_credential, _cred_write
-
-
 @pytest.mark.skipif(not sys.platform.startswith('win'), reason="This test only runs on Windows")
 def test_win_api():
-    service_name = u"VS Code test"
-    account_name = u"test"
+    from azure.identity._credentials.win_vscode_adapter import _read_credential, _cred_write
+    service_name = u"VS Code Azure"
+    account_name = u"Azure"
```

Comment:
Yes. For some reason, it failed in CI if I used "test". But given "Azure" is the one in real scenario. We are good as long as this one works fine.

Reply:
I think it would be better to know why the code is fragile. The adapter gets the account name from VS Code's settings, so it may not always be "Azure". And I still don't want a test that puts the Azure extension into an error state every time I run it.

Reply:
Fixed